### PR TITLE
RTT Veja Ravine to Overlong Bridge

### PR DESCRIPTION
**Pull request by ttgmichael**
Let me know whether I missed a typo or anything. :)

screenshot_20151011_00006

A note to @imcgames: I changed a line in the UI file from 'Rank' to 'Circle' which can be seen from the screenshot above. The line is shown in the Other Characters Screen in game and describes the circle # for the given character's current class. However in the Korean text it is shown as 랭크 or 'Rank', which I think ought to be 'Circle' in Korean. Would like to ask if we should change it to Circle to make it more accurate?

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -2185,8 +2185,8 @@ QUEST_LV_0100_20150317_002180	{memo X}The Mausoleum you should go can be reached
 QUEST_LV_0100_20150317_002181	It seems that rain won't stop. In fact, it rarely stopped.{nl}Do you think this is because of the evil energy?
 QUEST_LV_0100_20150317_002182	These thorny vines just keep growing even after we cut them.{nl}But it's better than doing nothing.
 QUEST_LV_0100_20150317_002183	Even if Bramble disappears, I think I am gonna stay like this for a while.{nl}The entire forest is covered with thorny vines. They are so big.
-QUEST_LV_0100_20150317_002184	So, you seem to live up to your reputation. {nl} Our people in the village are just cowards hoping the Goddess will take care of everything.
-QUEST_LV_0100_20150317_002185	But, did you not see the young man of our village? {nl} I told him to go see how the portal works, but I have not heard back since then.
+QUEST_LV_0100_20150317_002184	$You seem to live up to your reputation. {nl} The people in our village just cowardly hope the Goddess will take care of everything.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "are just cowards hoping"
or
"are just cowardly, hoping"

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -2185,8 +2185,8 @@ QUEST_LV_0100_20150317_002180	{memo X}The Mausoleum you should go can be reached
...
+QUEST_LV_0100_20150317_002185	$By the way, did you see a young man from our village? {nl} I told him to go check whether the portal works, but I have not heard back since then.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "check if the portal is working" ?

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -5763,11 +5763,11 @@ QUEST_LV_0100_20150317_005758	$Purified the Holy Pond, but the portal at Nugria
 QUEST_LV_0100_20150317_005759	$Check the Portal at Nugria Sanctum
 QUEST_LV_0100_20150317_005760	$Check the Portal at Nugria Sanctum.
 QUEST_LV_0100_20150317_005761	Defeat Moyabruka
-QUEST_LV_0100_20150317_005762	$Check the source of pollution in Vaidmens Uphill
-QUEST_LV_0100_20150317_005763	$Something is polluting the surroundings of Vaidmens Uphill. Defeat the source.
-QUEST_LV_0100_20150317_005764	The source of pollution is becoming stronger with the monsters. Defeat the monsters nearby and remove the source when it is weakened.
+QUEST_LV_0100_20150317_005762	$Check the source of corruption in Vaidma Uphill
+QUEST_LV_0100_20150317_005763	$Something is corrupting the surroundings of Vaidma Uphill. Defeat the source.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: Vaidma -> Veidma (x2)

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -6957,15 +6957,15 @@ QUEST_LV_0100_20150714_006952	$Be careful. This place is full of evil energy fro
...
 QUEST_LV_0100_20150714_006954	$There is a demon that I want you to defeat.{nl}This monster is called Rikaus and it's at Thorny Pillar Garden. It has been spreading an evil aura.
 QUEST_LV_0100_20150714_006955	$I heard a very ominous voice. It was a voice searching for you.{nl}Please be careful...
-QUEST_LV_0100_20150714_006956	$I never thought I would meet someone other than a follower in this place full of thorny vines and monsters.{nl}It's difficult without the protection of the Goddess. Don't you think so, Revelator?{nl}
-QUEST_LV_0100_20150714_006957	$So you are the Revelator. The followers are having a hard time due to the thorny vines.{nl}Can you help them, please?
+QUEST_LV_0100_20150714_006956	$I never thought I would meet someone other than a Believer in this place full of thorny vines and monsters.{nl}It's difficult without the protection of the Goddess. Don't you think so, Revelator?{nl}
+QUEST_LV_0100_20150714_006957	$Just as I thought. We believers are also having a hard time due to the thorny vines.{nl}Can you help us, please?
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "believers" -> "Believers" ?

File: ETC.tsv [hunk 5]
```diff
@@ -5158,7 +5158,7 @@ ETC_20150317_005157	Defeat Biteregina!
 ETC_20150317_005158	Small Hive
 ETC_20150317_005159	We are too late!
 ETC_20150317_005160	Wounded Villager
-ETC_20150317_005161	So you followed me here...
+ETC_20150317_005161	Were you looking for me?...
```
Comment (Sunaries) [inline ETC.tsv:5161]: This is completely different tone. I forgot what this was about, but _it_ doesn't really question if you were looking for him/her. He basically confirmed that you were following him.

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -2001,40 +2001,40 @@ QUEST_LV_0100_20150317_001996	Try to hold it in even if it's dirty. {nl} This is
 QUEST_LV_0100_20150317_001997	You found the revelation. {nl} Aren't you hurt? {nl}
 QUEST_LV_0100_20150317_001998	Bramble. {nl} Poor thing trying to go against its fate.
 QUEST_LV_0100_20150317_001999	$The Old Man of Andale Village
-QUEST_LV_0100_20150317_002000	$So you're the one who wanted to meet Goddess Saule. {nl} Do not worry about the pond. Our priests are working on it. {nl}
-QUEST_LV_0100_20150317_002001	$The priest is located at Cerpe Crossroads. {nl}I heard he needed the Black Maize Venom, so try to get him that as a gift.
+QUEST_LV_0100_20150317_002000	$So you're the one who wants to meet Goddess Saule. {nl} If you're here about the Pond, don't worry. Our priest is working on it. {nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:2004]: Is it necessary to capitalize it? It's in a dialogue, unless it mentions a specific map name, it should. 

I've talked with vtange about this. That in dialogue with an NPC, you should use lowercase unless it is really talking about a monster (name), NPC (name), map (name) or item (name). It's more natural to use lowercase in this case.

File: QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -6208,7 +6208,7 @@ QUEST_LV_0100_20150317_006203	Defeated Rikaus that was spreading evil aura. Repo
 QUEST_LV_0100_20150317_006204	Defeat Rikaus
 QUEST_LV_0100_20150317_006205	Talk to Believer Evaldas
 QUEST_LV_0100_20150317_006206	$Goddess Saule Believer Evaldas needs your help.
-QUEST_LV_0100_20150317_006207	$Defeat the Merog Shamans
+QUEST_LV_0100_20150317_006207	$Defeat the Merlog Shamans
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Why the change?

File: QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -6300,27 +6300,27 @@ QUEST_LV_0100_20150317_006295	{memo X}
...
 QUEST_LV_0100_20150317_006304	Collect the Languid Herbs.
-QUEST_LV_0100_20150317_006305	Let's create Languid Bomb to defeat monsters threatening Andale Village. Collect Languid Flowers need for making the bomb.
-QUEST_LV_0100_20150317_006306	$Talk to Andale Village Disciple
-QUEST_LV_0100_20150317_006307	Gathered enough Languid Flowers. Talk to the Village Disciple in Andale Village. 
-QUEST_LV_0100_20150317_006308	Ask the disciple what else is needed to make the Languid Bomb.
-QUEST_LV_0100_20150317_006309	Collect the strongly scented soul flower.
+QUEST_LV_0100_20150317_006305	Let's create a Languid Bomb to defeat monsters threatening Andale Village. Collect Languid Herbs needed to make the bomb.
+QUEST_LV_0100_20150317_006306	$Talk to the Andale Village Disciple
+QUEST_LV_0100_20150317_006307	$Gathered enough Languid Herbs. Talk to the Village Disciple in Andale Village. 
+QUEST_LV_0100_20150317_006308	Ask the Disciple what else is needed to make the Languid Bomb.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Necessary to capitalize this (along with Believer)? Because isn't 'disciple' a common noun? It would be different if it said 'Disciple X'.

Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Well if we are going to be capitalizing Guards and Soldiers; shouldn't we keep the same pattern for any NPC?

File: QUEST_LV_0100.tsv (hunk 7 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Most of the game text uses Merlog apparently, should I change every other line to Merog instead?

File: QUEST_LV_0100.tsv (hunk 6 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:2004]: The Pond mentioned is the Holy Pond; I believe it's ok to capitalize because the speaker is refering to none other than the Holy Pond, not any random pond.

File: ETC.tsv (hunk 5 above)
Comment (ttgmichael) [inline ETC.tsv:5161]: Except that this was the dialogue for the person you were looking for when the Elder said one of his villager didn't come back after checking whether the portal was working. The former line implies you followed him to where he was; which I find awkward because you never did follow him, you happened to only find him standing there.

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -1991,50 +1991,50 @@ QUEST_LV_0100_20150317_001986	Good job.{nl}I was worried that you would get hurt
...
+QUEST_LV_0100_20150317_002025	$That'll be plenty. {nl} I'll send the materials to the elderly, so please go and get the explosives.
 QUEST_LV_0100_20150317_002026	$Andale Village Headman
 QUEST_LV_0100_20150317_002027	$Explosives? Ahh. {nl} Go to the warehouse lot at the upper side of the village.
 QUEST_LV_0100_20150317_002028	$It's probably in the bigger containers. {nl} They won't explode, so don't worry.
 QUEST_LV_0100_20150317_002029	$Don't worry, it's safe. They will not explode.
 QUEST_LV_0100_20150317_002030	$It was in a small container? {nl} I must be getting old and mixed up. {nl} Normal people would have been incinerated, but a Revelator surely is different.
 QUEST_LV_0100_20150317_002031	{memo X}
-QUEST_LV_0100_20150317_002032	God or Goddess, what you call her doesn't matter. {nl} Aren't we all one eventually? {nl}
-QUEST_LV_0100_20150317_002033	Is it hard to understand? {nl} You will realize this one day.
+QUEST_LV_0100_20150317_002032	God or Goddess, Their title isn't really that important. {nl} Eventually, aren't we all just one being? {nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Their -> their

**Comment by Sunaries** [comment]
Good to merge :+1:

**Comment by imcgames** [comment]
:+1:
